2021/34xxx/CVE-2021-3432.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Expected location (2021/3xxx/CVE-2021-3432.json) of file does not match actual location (2021/34xxx/CVE-2021-3432.json)